applications/mimic_debug.cpp

```diff
@@ -0,0 +1,58 @@
+#include <libhal-armcortex/dwt_counter.hpp>
+#include <libhal-armcortex/startup.hpp>
+#include <libhal-armcortex/system_control.hpp>
+#include <libhal-lpc40xx/output_pin.hpp>
+#include <libhal-lpc40xx/system_controller.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal/steady_clock.hpp>
+
+#include "implementations/adc_mux_cd74hc4067.hpp"
+#include "implementations/degree_manipulation.hpp"
+#include "implementations/mission_control_handler.hpp"
+#include "implementations/utility.hpp"
+
+#include "hardware_map.hpp"
+
+using namespace hal::literals;
+using namespace std::chrono_literals;
+
+hal::status application(arm_mimic::hardware_map& p_map) {
+
+  const uint8_t N = 6;
+
+  while (true) {
+    auto digital_multiplexer = arm_mimic::AdcMuxCd74hc4067(p_map.adc_pin, p_map.signal_0, p_map.signal_1, p_map.signal_2, p_map.signal_3, p_map.steady_clock);
```

Comment:
The declaration should be outside of the while true or else your reinitializing it every time.